.travis.yml

```diff
@@ -2,6 +2,17 @@ language: ruby
 
 sudo: false
 
+cache:
+    apt: true
+    directories:
+        - $HOME/.cargo
+        - $HOME/.rustup
```

Comment:
I re-read the Travis caching docs recently (https://docs.travis-ci.com/user/caching/), a few questions:

1. Does `apt: true` actually do anything?
2. Do you want `cache: cargo`? So the `target` directory is backed up as well.

Reply:
1. Apparently not.
2. I *think* if I'm reading this correctly "cache: cargo" is only available if you're using language: rust (their example doesn't use bundler: true when caching rubygems under rust).

Also although caching target is a good idea it runs into problems with stale builds because of stupid issues with the Helix rake task not actually exiting non-zero when the build fails because why would you want that? 🙄